cmd/get.go

```diff
@@ -119,6 +126,10 @@ func (o *downloadOption) preRunE(cmd *cobra.Command, args []string) (err error)
 	}
 	o.URL = targetURL
 
+	if o.ProxyGitHub != "" {
+		o.URL = strings.ReplaceAll(o.URL, "github.com", fmt.Sprintf("%s/github.com", o.ProxyGitHub))
```

Comment:
Please use `strings.Replace` instead. Consider that url is equal to `https://github.com/johnniang/github.com/xxx`.